src/DetailsView/components/tab-stops-failed-instance-section.tsx

```diff
@@ -39,11 +42,14 @@ export const TabStopsFailedInstanceSection = NamedFC<TabStopsFailedInstanceSecti
             if (data.status !== 'fail') {
                 continue;
             }
+            const displayAutmatedInfo =
```

Comment:
nit (because this is going away after feature flag is removed): minor typo, missing the o in `displayAutmatedInfo`